### PR TITLE
TDatD: Endless EULA

### PR DESCRIPTION
**Pull request by msbeck**
I used Pup as an archetype for Endless EULA because it has subroutines which do not fire if the runner pays a credit.

```
   "Pup"
   {:subroutines [(do-net-damage 1)]
    :runner-abilities [(runner-break [:credit 1] 1)]}
```

### CODE REVIEW

File: src/clj/game/cards/ice.clj [hunk 1]
```diff
@@ -723,6 +723,12 @@
                   end-the-run]
     :runner-abilities [(runner-break [:click 2] 2)]}
 
+   "Endless EULA"
+   {:implementation "Encounter effect is manual. Runner choice is not implemented"
```
Comment (NoahTheDuke) [inline src/clj/game/cards/ice.clj]: Maybe reword as, "Runner choice is manual."

**Comment by msbeck** [comment]
Yes, I can update the text to more appropriately reflect the runner choice. I pulled this from Data Raven which is a different occasion.

**Comment by msbeck** [comment]
I just played a game where it said Endless EULA was "unimplemented"
